layouts/partials/css.html

```diff
@@ -21,34 +21,33 @@
 <!-- Fallback font for symbols (such as "🛈"). --
 <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Noto+Sans+Symbols+2">
 
-<!-- SASS -->
-{{- $sass := (slice "theme-css/pst/pydata-sphinx-theme.scss") | append (resources.Match "css/*.scss") -}}
+<!-- Process and include Sass files. -->
+{{- $sass := (slice "theme-css/pst/pydata-sphinx-theme.scss")
+           | append (resources.Match "theme-css/*.scss")
```

Comment:
Presumably these changes now happen in another PR.

Reply:
Yes, in https://github.com/scientific-python/scientific-python-hugo-theme/pull/429

Reply:
Merged, so this can be rebased.